scripts/www/build-cfg-help.pl

```diff
@@ -247,9 +247,12 @@ ()
     }
 }
 
+my $linecount = 0;
```

Comment:
If you have not already, please consider using `$.` (or equivalent) instead of maintaining your own line count. See "man perlvar" for more details, including `@ARGV` handing that is happening in the loop below:

```
$. is reset when the filehandle is closed, but not when an open filehandle is reopened without an intervening "close()".  For
more details, see "I/O Operators" in perlop.  Because "<>" never does an explicit close, line numbers increase across "ARGV"
files (but see examples in "eof" in perlfunc).
```

Reply:
Done in [ae14820](https://github.com/squid-cache/squid/pull/1546/commits/ae148203f6b1b2afb2497bc48191d21fbdad8be5)